src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String taskCode(){
+        return "";
+    }
+
+    public String getTime(){
+        //returns nothing here cause there is no time component
+        return "";
+    }
```

Comment:
It seem strange that the taskcode is blank

And that getTime returns blank also. Perhaps you might need to tweak your hierarchy such that it each class only contains the fields it needs.